model/qti/AssetParser.php

```diff
@@ -356,6 +359,17 @@ private function getXmlProperties($properties)
         return $xmls;
     }
 
+    private function extractAdvancedCustomInteractionAssets(CustomInteraction $interaction): void
+    {
+        $assetExtractor = ExtendedCustomInteractionAssetExtractorAllocator::allocateExtractor($interaction);
+        if ($assetExtractor !== null) {
```

Comment:
If you decide not to implement a no-op service here, could you please consider adding an early `return` instead of a logic with an `if` block?
That improves readability, keeping the indentation not too nested.

Reply:
i will create no-op service